specs/spec-1-kudos.md

```diff
@@ -1,107 +1,269 @@
----
-#: 1
-Status: Review
-Category: NEAR Social
-Created: 2023-04-05
-Replaces: --
-Requires: --
----
+<!-- Objective. In a few sentences, describe the key system objectives. Tweet length executive summary. -->
+# RFP 2: Kudos Widget
+Kudos is a widget to build a reputation. The widget will allow any verified human to nominate, recieve, and interact with Kudos on Near Social.
+
+# Metadata
+
+<!-- Common status: Inception, Draft, Community consultation, Locked: RFP ongoing, Locked: RFP awarded, Obsolete -->
+## Status
+| Key       | Value                 |
+|-----------|-----------------------|
+| **#**    | 1                     |
+| **Status**  | Review              |
+| **Category** | NEAR Social        |
+| **Replaces** | --                 |
+| **Requires** | --                 |
+
+<!--
+Points for reviewers to consider:
+    - Be pragmatic: Is there enough information to begin coding? Practically, implementation will reveal the unforeseeable. This doc can evolve to record those issues/resolutions.
+    - Wide angle view: Do you know of overlapping projects? Are there OKRs or other non-technical forces that would push/pull priority?
+    - Optimization: Describe alternate approaches and why they are better. Use Caveats section for approaches under consideration, even if they are ultimately abandoned. 
+--> 
+## Changelog
```

Comment:
Why changing the order

Reply:
Which order?

Reply:
Any suggestions on solving the [markdown-link-check error](https://github.com/near-ndc/gwg/actions/runs/4855163900/jobs/8653399493?pr=4)? Doesn't seem related to the Kudos spec.